Tropos/Models/DailyForecast.swift

```diff
@@ -1,10 +1,10 @@
 import Foundation
 
-@objc(TRDailyForecast) class DailyForecast: NSObject {
-    let date: NSDate
-    let conditionsDescription: String
-    let highTemperature: Temperature
-    let lowTemperature: Temperature
+struct DailyForecast {
+    var date: NSDate
+    var conditionsDescription: String
+    var highTemperature: Temperature
+    var lowTemperature: Temperature
```

Comment:
Main reason is it's more flexible, and allows you to use the type with `var` in the expected way. It's still safe because of value semantics.